src/modules/Dashboard/components/ImagesCardView/ImageCard/Controls/controls.scss

```diff
@@ -3,37 +3,32 @@
 .controls {
 	display: flex;
 	align-items: center;
-	justify-content: flex-end;
-	margin-top: 10;
+	justify-content: center;
+	padding: 10px 0;
 }
 
 .control {
+	line-height: 13px;
 	display: flex;
 	flex-direction: column;
 	align-items: center;
-	justify-content: center;
-	font-size: 12;
+	justify-content: space-between;
+	height: 37px;
```

Comment:
why you are using a non-multiplied values? Eugene says our multiplier is 4, as i remember.